test/controllers/CheckYourAnswersControllerSpec.scala

```diff
@@ -444,8 +445,8 @@ class CheckYourAnswersControllerSpec extends SpecBase with MockitoSugar with Bef
               val expectedRegistrationRequest = OrganisationWithoutId("name", Address.apply(aBusinessAddress), aContactDetails)
               val expectedContact = OrganisationContact(models.subscription.Organisation("contact name"), aContactDetails.emailAddress, None)
               val expectedSubscriptionRequest = SubscriptionRequest("safeId", false, None, expectedContact, None)
-              val subscriptionResponse = SubscribedResponse("dprsId")
-              val subscriptionDetails = SubscriptionDetails(subscriptionResponse, expectedSubscriptionRequest, RegistrationType.ThirdParty, Some(BusinessType.LimitedCompany))
+              val subscriptionResponse = SubscribedResponse("dprsId", Instant.now())
+              val subscriptionDetails = SubscriptionDetails(subscriptionResponse, expectedSubscriptionRequest, RegistrationType.PlatformOperator, Some(BusinessType.LimitedCompany), Some("name"))
```

Comment:
Perhaps creating SubscriptionDetailsBuilder will be useful to generate default SubscriptionDetails for testing.